docs/blog/2018-07-07-graphic-design-class/index.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Learning how to code with Gatsby
+date: "2018-07-07"
+author: "Shannon Soper"
+tags: ["college", "case study"]
+---
+
+## Phil Howley's Graphic Design class
+
+Recently, a Gatsby fan named Phil Howley starting teaching Gatsby in an Emerging Technologies class at the Art Institute (AI) in North Hollywood and I got to interview him.
+
+In his 10 years of teaching over 50 web development/design classes at (AI) (his other job is with [https://ironcovesolutions.com](https://ironcovesolutions.com) which is built on Gatsby!), he’s observed that graphic design students tend to come to class less as the class gets more focused on coding.
+
+However, in his current class _nobody has dropped out_. And every single one of his students turned in a portfolio, which is also unheard of in this particular class. The reason? He taught them to create their portfolios using Gatsby.
+
+## Here are some of the student’s repos and final sites:
+
+![Image of graphic designer's site](graphic-design-1.png)
```

Comment:
Can we format these this way:

[Site Title](http://site-url) ([source](http://github-repo))

and also make the image link to the site?